automation/common.sh

```diff
@@ -84,6 +94,7 @@ runapi () {
 		-e SUPERUSER_PASSWORD=foobarbaz\
 		$extra_vol_args \
 		$extra_env_args \
+		-p 9229:9229 \
```

Comment:
There's a `--profile` option already in at least `fasttest` for exposing the inspector port, and it also enables the debugger/profiler at the same time

Reply:
Yeah I have been using `--profile` but it did not seem to open a port on `9229`. Now taking a closer look I see I should have been attaching to `9228` `api_id=$(runapi $IMAGE_NAME $db_id $redis_id $loki_id " -p 9228:9229 -v $(pwd):/usr/src/app")`. is the port change intentional?

Removed `-p 9229:9229 \`.

Reply:
Ah yeah, that was to help avoid clashes with other debugger ports that might be open